saga/sagaState.go

```diff
@@ -5,28 +5,28 @@ import (
 	"fmt"
 )
 
+type flag byte
```

Comment:
(?) ANDing bitflags will always produce zero.

This is a list of all the messages received for this task(START, END, STARTCOMP, ENDCOMP)
they don't always all happen in a Saga.  i.e. if there is an END there is always a START, but if there is a STARTCOMP there isn't always an END message.  See the updateFunction for full rules.